src/main/g8/src/main/resources/local-shared.conf

```diff
@@ -21,12 +21,11 @@ akka.discovery.config.services {
   "$name;format="word-only,lowercase"$" {
     endpoints = [
       {host = "127.0.0.1", port = 9101},
-      {host = "127.0.0.1", port = 9102}
+      {host = "127.0.0.1", port = 9102},
+      {host = "127.0.0.1", port = 9103}
     ]
   }
 }
 
 # for reduced Projection latency
 akka.persistence.cassandra.events-by-tag.eventual-consistency-delay = 200 ms
-
-
```

Comment:
No idea why this two empty lines were removed. Locally I see there is one blank like on lnum 32